.github/workflows/deploy.yml

```diff
@@ -0,0 +1,31 @@
+# This workflows will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
+
+jobs:
+
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.5'
```

Comment:
We build the package with python 3.5, but we run the tests only with python versions 3.6 3.7 and 3.8. Either we should change this version here to use a tested one or we should add 3.5 to the test matrix.

Reply:
Good point. PR is updated.